content/docs/learning-guide/tutorials/_index.md

```diff
@@ -24,3 +24,9 @@ Learn the features, tools, and processes of **Open 3D Engine (O3DE)** with the h
 | Tutorial | Description |
 | - | - |
 | [Your First Network Component](multiplayer/first-multiplayer-component) | Create multiplayer components with C++ in this introduction to network components. |
+
+## PostFX
+
+| Tutorial | Description |
+| - | - |
+| [PostFX Shape Weight Modifier](postfx/use-postfx-shape-weight-modifier) | Use PostFX Shape Weight Modifier to modify exposure control in O3DE. |
```

Comment:
A couple of suggestions: 
- Specify "PostFX Shape Weight Modifier component"
- Describe a general application users can learn from this tutorial. (They'll learn how to modify PostFX in general, and this example demonstrates that by modifying the camera exposure.)
```suggestion
| [PostFX Shape Weight Modifier](postfx/use-postfx-shape-weight-modifier) | Use a **PostFX Shape Weight Modifier** component to modify exposure control in O3DE. This example demonstrates how to modify post-processing effects (PostFX) at runtime. |
```